### PR TITLE
Another fix to test out

### PR DESCRIPTION
**Pull request by ivanaveliskova**

### CODE REVIEW

**Review by Jacfem — COMMENTED** [review]
Taking a look, thanks Ivana -- if/before we merge this, should we also remove the changes done from https://github.com/guruhq/slate-edit-table/pull/6/files since we found some bugs with that after merging + bumping this package version to 0.13.2?

File: dist/validation/schema.js [hunk 1]
```diff
@@ -99,9 +99,9 @@ function onlyBlocksInCell(opts, change, context) {
 function noBlocksWithinCell(opts, change, context) {
   if (context.child.object === "block") {
     if (context.child.isVoid) {
-      return change.unwrapNodeByKey(context.child.key);
+      return change.removeNodeByKey(context.child.key, { normalize: false });
```
Comment (Jacfem) [inline dist/validation/schema.js]: Is this essentially removing the added paragraph tags if its child is void, to try making the tables more performant? I tried running the slate-tools branch locally and from my nightmare card created on the earlier version of slate, things look good, but I'll create some that have things inside of tables to help us test
https://qaapp.getguru.com/card/Tnnbjgac/Nightmare-Card-from-previous-slate-version (except I tested it locally)

Comment (ivanaveliskova) [reply dist/validation/schema.js]: Yeah we noticed this was essentially backwards. The context should be the table cell, we are checking if the child is a block, if so then we are checking if it's void, like a divider, we don't want those so we remove them (also void won't have any text in it), while we want to keep any text, so we unwrap the paragraph block. We noticed it was backwards when doing some testing last night.

**Comment by ivanaveliskova** [comment]
> Taking a look, thanks Ivana -- if/before we merge this, should we also remove the changes done from https://github.com/guruhq/slate-edit-table/pull/6/files since we found some bugs with that after merging + bumping this package version to 0.13.2?

We want to keep those because this builds upon them. We are going back to having only inlines inside of tables and so this just fixes additionally what we changed in that version

**Comment by Jacfem** [comment]
Awesome, sounds good + just wanted to check 👍

File: dist/validation/schema.js (hunk 1 above)
Comment (Jacfem) [reply dist/validation/schema.js]: Great, that makes sense. That's so interesting

**Review by Jacfem — APPROVED** [review]

**Pull request merged**